app/jinja_filters.py

```diff
@@ -98,3 +132,8 @@ def format_number_to_alphabetic_letter(number):
 @blueprint.app_context_processor
 def utility_processor():
```

Comment:
Name updated, so that it reflects what its doing. I've changed the method format_condtional_date to accept two dates. Tests revised in consultation with Andrew.